deny.toml

```diff
@@ -120,6 +120,7 @@ allow = [
 
     # Font licenses.
     "Unicode-DFS-2016",     # https://spdx.org/licenses/Unicode-DFS-2016.html
+    "Unicode-3.0",          # https://spdx.org/licenses/Unicode-3.0.html
```

Comment:
Do you know which crate is pulling this in?

Reply:
heed pulls in `url` which pulls in crates from here: https://github.com/unicode-org/icu4x